### PR TITLE
added steps for gpu config in readme

### PR DESCRIPTION
**Pull request by Anirudh58**
Improved readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,13 +61,18 @@ Please refer to [official doc](https://docs.docker.com/engine/install/).
...
 ```
-3. Below is a basic query that should work on the client
+
+3. Run the `LOAD` command in the client terminal: (may take a while)
+```mysql
+LOAD DATA INFILE 'test_video.mp4' INTO MyVideo;
+```
+4. Below is a basic query that should work on the client
 ```mysql
 SELECT id, data FROM MyVideo WHERE id < 5;
```
Comment (gaurav274) [inline README.md:77]: Personally, I feel this query doesn't look good on the readme. It doesn't accomplish much. How about we replace it with the select query in tutorial? At least that has object detection. What do you think?

**Review by gaurav274 — COMMENTED** [review]
LGTM. Why isn't the testsuite getting triggered?

File: README.md (hunk 1 above)
Comment (Anirudh58) [reply README.md:77]: But when a person is starting off, they need to know how to load a video right? for any select query to even work.

Comment (gaurav274) [reply README.md:77]: Ah! I missed it is under the client testing. Then, it is fine.

**Review by gaurav274 — APPROVED** [review]

**Pull request merged**